cmd/dep/testdata/harness_tests/init/manifest-exists/final/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Your right i think I can remove it when i get the non existing file issue sorted.